orchard_note_encryption.py

```diff
@@ -252,17 +252,18 @@ def randbytes(l):
         assert(memo_using_ivk == memo)
 
         test_vectors.append({
+            'incoming_viewing_key': receiver_fvk.dk + bytes(ivk),
```

Comment:
Will existing tests have to split this value into its parts? If so, would it be better to emit them as separate fields?